tests/app/main/views/test_services.py

```diff
@@ -239,6 +221,29 @@ def test_service_view_shows_info_banner_for_removed_and_private_services(self, d
             )
         ]
 
+    @pytest.mark.parametrize('service_status', ['disabled', 'enabled'])
+    def test_info_banner_contains_publish_link_for_ccs_category(self, data_api_client, service_status):
+        self.user_role = 'admin-ccs-category'
```

Comment:
We don't need to set this user_role per test as it's set for the class as `admin-ccs-category` at line 40, so you only need to set `self.user_role` for tests that need something other than Category.

Reply:
Will change

Reply:
Ah that was brought in by the rebase if I'm reading the diff right. Thought I'd checked that. I've checked and where necessary updated all the classes in that file in a new commit.